astro/src/content/docs/release-notes/index.mdx

```diff
@@ -106,6 +106,7 @@ Update dependencies.
 * Applications now offer an `Unknown Scope Policy`. This can be used to enhance security by rejected or removing unrecognized scopes during an OAuth workflow. See the application [Scopes tab](/docs/get-started/core-concepts/applications#scopes) for more detail.
   * Delivered as part of the Custom OAuth Scopes body of work, which resolves [GitHub Issue #275](https://github.com/FusionAuth/fusionauth-issues/issues/275) (see below)
 * Applications now have a new Scope Handling Policy. The `Strict` option provides behaviors that are more compliant with the OIDC specification, while the `Compatibility` option provides backwards-compatible behavior. Specifically, `Strict` mode limits information in access tokens and populates Id tokens and UserInfo responses based on the requested OAuth scopes. This option also restricts the UserInfo endpoint to accepting only access tokens containing the `openid` scope.
+Note, this change can affect new integrations as any new application will default to `Strict` (please specify a `Compatibility` field value in your API calls if needed).
```

Comment:
I would make this a new bullet point above the _Resolves GitHub Issue_ bullet. I think this reads a little bit more like other release notes.
> Note that new applications will default to the `Strict` option. If your integration requires the `Compatibility` policy, please specify that option when creating the application.

Reply:
(comment from the peanut gallery) you can usually remove 'Note' and improve impact:l

>  New applications will default to the `Strict` option. If your integration requires the `Compatibility` policy because they need backwards compatible behavior, specify that option when creating the application.